.github/workflows/api-docs-test.yml

```diff
@@ -43,23 +55,17 @@ jobs:
       - name: Setup sentry python env
         uses: ./.github/actions/setup-sentry
         id: setup
+        if: steps.changes.outputs.api_docs == 'true'
         with:
           python: 3
           snuba: true
 
-      - name: yarn cache
-        uses: actions/cache@v2
-        id: yarn-cache # use this to check for `cache-hit` (`steps.yarn-cache.outputs.cache-hit != 'true'`)
-        with:
-          path: ${{ steps.setup.outputs.yarn-cache-dir }}
-          key: ${{ runner.os }}-yarn-${{ hashFiles('**/yarn.lock') }}
-          restore-keys: |
-            ${{ runner.os }}-yarn-
-
```

Comment:
I just removed this as it's not really necessary... although I may want to followup and change what we're hashing... we should only hash the main `yarn.lock` file

Reply:
Why? Isn't it an easy change? Instead of `**/yarn.lock`, you want `api-docs/yarn.lock`.

Reply:
`yarn install` currently takes 5s, doesn't seem worth the overhead?

Reply:
Oh, I see.